cosmoz-treenode.js

```diff
@@ -151,6 +151,7 @@ class CosmozTreenode extends PolymerElement {
 	}
 
 	_computePathText(pathToRender, valueProperty, pathSeparator) {
+		console.log('test eslint error');
```

Comment:
To be removed?

Reply:
Yes, tried to test the annotations as seen https://github.com/gimenete/eslint-action  and https://github.com/rkusa/eslint-action